### DISCUSSION TITLE
Help with OUPath field on AD join

### DISCUSSION THREAD

**Discussion by shettys-ops** [hashicorp/packer-plugin-vsphere#640] [Community Help]
Hi Team,

I'm currently working on provisioning using the vsphere_virtual_machine resource with a customized template (created using Packer). With the customize - windows_options option, I'm able to add the newly provisioned VM to the domain, but it lacks the OU Path argument.

Could you please assist me in finding a way to add the machines to the correct OU? Any workarounds would also be appreciated

**Comment by tenthirtyam** [comment]
Are you cloning the image crated with Packer and then trying to add the new VMs (from the template) to AD using Terraform? If so, this would be related to the Terraform provider, not the Packer plugin.

**Comment by tenthirtyam** [comment]
[hashicorp/terraform-provider-vsphere #2130](https://github.com/hashicorp/terraform-provider-vsphere/issues/2130?notification_referrer_id=NT_kwDOAHaU47I5NDQwMzQzNjA4Ojc3NzEzNjM#issuecomment-1939559490)
